### PR TITLE
#3613 #3614 The Mobile preview switcher tooltip always shows 'Landscape'

### LINKED ISSUE #3613: The Mobile preview switcher tooltip always shows 'Landscape'
Run the preview and enable mobile mode for the creator ([demo](https://surveyjs.io/survey-creator/examples/free-nps-survey-template/reactjs)). Observe the orientation switcher always shows 'Landscape'. It should be changed to 'Portrait' / 'Landscape'
image

### LINKED ISSUE #3614: A Creator option/setting to define the default Mobile Preview Orientation
The Creator's mobile preview always starts in Landscape mode:
image

To change the default orientation, it is necessary to disable the `simulator.landscape` option ([example](https://plnkr.co/edit/IvFh7s0qyZW1lAWE))
```
creator.onActiveTabChanged.add((sender, options) => {
  if(options.tabName === "test") {
    const simulator = options.model.simulator;
    simulator.landscape = false;
  }
});
```
image

Suggestion: create a Creator / CreatorOptions setting which will define the default Mobile Preview orientation.

### PR DESCRIPTION
**Pull request by novikov82**
Fixes #3613 Fixes #3614

### CODE REVIEW

File: packages/survey-creator-core/src/creator-options.ts [hunk 1]
```diff
@@ -216,10 +216,16 @@ export interface ICreatorOptions {
    */
   allowChangeThemeInPreview?: boolean;
   /**
-   * Limits question and panels types available in the Toolbox and Add Question menu.
+   * Limits question and panels types available in the Toolbox and Add Question menu.
```
Comment (novikov82) [inline packages/survey-creator-core/src/creator-options.ts:219]: need description

File: packages/survey-creator-core/src/creator-base.ts [hunk 2]
```diff
@@ -222,6 +222,12 @@ export class CreatorBase extends Base
   @property({ defaultValue: false }) showSearch: boolean;
   @property({ defaultValue: true }) generateValidJSON: boolean;
   @property({ defaultValue: "" }) currentAddQuestionType: string;
+  /**
+   * ?????
+   *
+   * Default value: `landscape`
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/creator-base.ts:228]:
```suggestion
   * Specifies the orientation of the selected device in the Preview tab.
   *
   * Possible values:
   * - `"landscape"` (default)
   * - `"portrait"`
```

File: packages/survey-creator-core/src/creator-options.ts [hunk 3]
```diff
@@ -216,10 +216,16 @@ export interface ICreatorOptions {
    */
   allowChangeThemeInPreview?: boolean;
   /**
-   * Limits question and panels types available in the Toolbox and Add Question menu.
+   * Preview orientation.
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/creator-options.ts:219]:
```suggestion
   * Specifies the orientation of the selected device in the Preview tab.
```

File: packages/survey-creator-core/src/creator-options.ts [hunk 4]
```diff
@@ -216,10 +216,16 @@ export interface ICreatorOptions {
    */
   allowChangeThemeInPreview?: boolean;
   /**
-   * Limits question and panels types available in the Toolbox and Add Question menu.
+   * Preview orientation.   
    * 
-   * Refer to the [Limit Available Question and Panel Types](https://surveyjs.io/Documentation/Survey-Creator?id=toolbox#limit-available-question-and-panel-types) help topic for more information.
-   */
+   * Default value: "landscape"
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/creator-options.ts:221]:
```suggestion
   * Possible values:
   * - `"landscape"` (default)
   * - `"portrait"`
```

**Review by tsv2013 — APPROVED** [review]

**Pull request merged**